app/controllers/mentors_controller.rb

```diff
@@ -0,0 +1,4 @@
+class MentorsController < ApplicationController
+  def index
+  end
+end
```

Comment:
check it out.. we can totally fake this data. we know that we are building a API in the backend. this API will return JSON for us.. it will always return JSON.. so this code might look like this..
```ruby
  def index
    @mentors = 15.times.map { |_| mentors_payload }
    #@mentors = nil
  end


  private
  def mentors_payload
    {
      "avatar": "user.jpg",
      "name": Faker::Name.name,
      "company": Faker::Company.name,
      "specialty": Faker::Job.field,
      "email": Faker::Internet.email,
      "phone": Faker::PhoneNumber.cell_phone
    }.as_json
  end
```

Reply:
good idea, I will implement this